addons/controllers/testdata/test-tkg-system-ns-resources.yaml

```diff
@@ -174,6 +178,8 @@ kind: KappControllerConfig
 metadata:
   name: test-cluster-tcbt-kapp-controller-config
   namespace: tkg-system
+  annotations:
+    tkg.tanzu.vmware.com/template-config: "true"
```

Comment:
why were this not here before? was it just an omittion or are this supposed to be non-template configs. 
From the rest of the code it seem they are supposed to be so they need to have the  correct label